libs/ui/src/lib/record-preview-title/record-preview-title.component.html

```diff
@@ -1,6 +1,6 @@
 <!-- Record list item: List Small -->
 <div
-  class="h-10 border-b border-gray-200 transition duration-200 hover:bg-gray-100 border-gray-300"
+  class="h-10 w-full bg-white border border-gray-300 transition duration-200 hover:bg-gray-100 rounded-sm"
```

Comment:
I think just bottom border, very light is enough, full border is too much for this layout

Reply:
Ah les goûts et les couleurs - Consensus will be a challenge ;)

That's a difficulty/challenge I see with tailwind, you define a style somewhere eg. border-width and color in HTML. You expect to have some kind of consistencies all hover the places (even maybe all hover the apps) but that needs to be maintained. We used to define a css class gn-results and style the border in CSS/LESS file. Here we are defining the border in each layout - you can create a shortcut for a custom tailwind rule or create your themes in tailwind.config.js overriding the meaning of rounded-sm. Not sure what is better...

Reply:
Yes, it gives you a lot of freedom 😄

Reply:
> We used to define a css class gn-results and style the border in CSS/LESS file

True that defining our own classes that reference TW classes could have been great to homogeneize the theme.
How ever it is doable with less but not with css.

> you can create a shortcut for a custom tailwind rule or create your themes in tailwind.config.js overriding the meaning of rounded-sm

ok, let's see how it works. @MichelGabriel did you set up something like this in the PR or not ?

Reply:
> Ah les goûts et les couleurs - Consensus will be a challenge ;)

It's just that in EEA forest thing, the news list has no border, and in the TITLE card you have done, I only see a border at the bottom ... while I don't see any specific class for this (seems your HTML declare a full border).

Also @MichelGabriel I would rather go for a margin in list items themselves rather than a padding in item container, actually it gives the freedom to have different spacing depending on the CARD mode.

Reply:
> the news list has no border, 

hum,
![image](https://user-images.githubusercontent.com/1701393/102872164-a5c96600-443f-11eb-86b6-3072634e91fb.png)


> and in the TITLE card you have done,

I know it's my fault.

> I only see a border at the bottom ... while I don't see any specific class for this (seems your HTML declare a full border).

See
https://github.com/geonetwork/geonetwork-ui/blob/master/libs/ui/src/lib/record-preview-title/record-preview-title.component.html#L3 `border-b` means border bottom.

Reply:
> Also @MichelGabriel I would rather go for a margin in list items themselves rather than a padding in item container, actually it gives the freedom to have different spacing depending on the CARD mode.

This will probably give problems when you want to change the background color of a list item.

Reply:
@fxprunayre What should the TITLE list look like? A border for the whole list?

Reply:
@MichelGabriel the image just above

Reply:
But that has only the border at the bottom, so that's what I did, or not?

Reply:
> you can create a shortcut for a custom tailwind rule or create your themes in tailwind.config.js overriding the meaning of rounded-sm

What does creating a shortcut mean exactly?

Reply:
> But that has only the border at the bottom, so that's what I did, or not?

When I tried the first time, the TITLE card had a full border. Indeed you removed `border-b` class.
My thought was just that for a very small list like TITLE, a full border is a little bit cumbersome